build.gradle.kts

```diff
@@ -0,0 +1,96 @@
+plugins {
+    kotlin("multiplatform") version "1.9.0"
+    `maven-publish`
+    signing
+    id("io.github.gradle-nexus.publish-plugin") version "1.3.0"
+    id("io.freefair.lombok") version "8.3"
+    id("org.jetbrains.dokka") version "1.9.0"
+}
+
+repositories {
+    mavenCentral()
+}
+
+group = "org.hisp.dhis.lib.expression"
+version = "1.1.0-SNAPSHOT"
+
+val isReleaseVersion = project.hasProperty("removeSnapshot")
+if (isReleaseVersion) {
+    version = (version as String).replace("-SNAPSHOT", "")
+}
```

Comment:
This part of the code is the replacement for the parameter `removeSnapshot` in `mvn -B versions:set -DremoveSnapshot -DgenerateBackupPoms=false`. If I understood it correctly, this parameter only affects the current publication and it doesn't commit the version without the snapshot. 
This piece of code has the same purpose. When the Gradle command includes `-PremoveSnapshot`, the build is done as if the version doesn't contain the suffix `-SNAPSHOT`.